test/v2/integration/trading.ts

```diff
@@ -0,0 +1,45 @@
+import { coroutine as co } from 'bluebird';
+import should = require('should');
+import * as nock from 'nock';
+
+const TestV2BitGo = require('../../lib/test_bitgo');
+
+describe('V2 Trading', function() {
+  let bitgo;
+  let basecoin;
+  let wallets;
+  let wallet;
+  let trading;
+
+  before(co(function *() {
+    nock.restore();
+
+    bitgo = new TestV2BitGo({ env: 'dev', microservicesUri: 'https://app.bitgo-dev.com' });
+    bitgo.initializeTestVars();
+    basecoin = bitgo.coin('ofc');
+    wallets = basecoin.wallets();
+    basecoin.keychains();
+
+    yield bitgo.authenticateOfcTestUser(bitgo.testUserOTP());
+    wallet = yield wallets.getWallet({ id: TestV2BitGo.V2.OFC_TEST_WALLET_ID });
+    trading = wallet.trading();
+  }));
+
+  it('should create and sign a trade payload', co(function *() {
```

Comment:
Seems like this test is simple enough that it could be turned into a unit test with the server nocked.